lighthouse-core/gather/driver.js

```diff
@@ -1081,7 +1088,7 @@ function wrapRuntimeEvalErrorInBrowser(err) {
  * property on window to the end time of the last long task.
  * instanbul ignore next
  */
-function installPerformanceObserver() {
+function registerPerformanceObserver() {
```

Comment:
how about `registerPerformanceObserverInPage` or something to signify the different context?